### ISSUE TITLE
[1.12.2] Geolosys retrogen and UB

### ISSUE THREAD

**Issue by TheConception** [Aang23/UndergroundBiomesConstructs#196]
Hello,

On my setup I am using Geolosys retrogen option which changes every mod generated ore into Geolosys one for added immersion.
The problem is that Geolosys applies its retrogen after everything, so different stone overlays from UB break and they show up as default stone ores in any vein.
Is there a way to queue UB overlays to be generated last? Or maybe some edits I can do in ores config?

Maybe this is related to Geolosys repo?

tl;dr:
mod generates ore -> UB overlays -> geolosys retrogen -> everything in geolosys ore, but with default stone overlay (sad face)

mod generates ore -> geolosys retrogen -> UB overlays -> everything in geolosys ore with proper UB overlays (wish)

**Comment by Aang23** [comment]
The event UBC replaces stone on is getting changed in the next release and most issues of that kind including villages should work as well as it does in 1.14.x+
